kit/ForKit.cpp

```diff
@@ -150,6 +150,12 @@ class ServerWSHandler final : public WebSocketHandler
                 LOG_WRN("Cannot spawn " << tokens[1] << " children as requested.");
             }
         }
+        else if (tokens.size() == 2 && tokens.equals(0, "setloglevel"))
+        {
+            // Set environment variable so that new children will also set their log levels accordingly.
+            setenv("LOOL_LOGLEVEL", tokens[1].c_str(), true);
```

Comment:
The second param to setenv is an int, not a bool (it's a C function). Best not to mix types (mostly to avoid confusing maintainers).

Reply:
Done.